src/xo-app/pool/tab-network.js

```diff
@@ -77,15 +85,93 @@ class PifItem extends Component {
   }
 }
 
+class PifsItem extends BaseComponent {
+  render () {
+    const { network } = this.props
+    const { showPifs } = this.state
+
+    return <div>
+      <Tooltip content={showPifs ? _('hidePifs') : _('showPifs')}>
+        <Button bsSize='small' bsStyle='secondary' className='m-b-1 pull-xs-right' onClick={this.toggleState('showPifs')}>
+          <Icon icon={showPifs ? 'hidden' : 'shown'} />
+        </Button>
+      </Tooltip>
+      {showPifs && <table className='table'>
+        <thead className='thead-default'>
+          <tr>
+            <th>{_('pifDeviceLabel')}</th>
+            <th>{_('homeTypeHost')}</th>
+            <th>{_('pifVlanLabel')}</th>
+            <th>{_('pifAddressLabel')}</th>
+            <th>{_('pifMacLabel')}</th>
+            <th>{_('pifStatusLabel')}</th>
+            <th />
+          </tr>
+        </thead>
+        <tbody>
+          {map(network.PIFs, pifId => <PifItem key={pifId} id={pifId} />)}
+        </tbody>
+      </table>}
+    </div>
+  }
+}
+
+const NETWORKS_COLUMNS = [
+  {
+    name: _('poolNetworkNameLabel'),
+    itemRenderer: network => <Text value={network.name_label} onChange={value => editNetwork(network, { name_label: value })} />,
+    sortCriteria: network => network.name_label
+  },
+  {
+    name: _('poolNetworkDescription'),
+    itemRenderer: network => <Text value={network.name_description} onChange={value => editNetwork(network, { name_description: value })} />,
+    sortCriteria: network => network.name_description
+  },
+  {
+    name: _('poolNetworkMTU'),
+    itemRenderer: network => network.MTU
+  },
+  {
+    name: <div className='text-xs-center'>
+      <Tooltip content={_('defaultLockingMode')}>
+        <Icon size='lg' icon='lock' />
+      </Tooltip>
+    </div>,
+    itemRenderer: (network, vifsByNetwork) => {
+      const networkInUse = some(vifsByNetwork[network.id], vif => vif.attached)
+      return _toggleDefaultLockingMode(
```

Comment:
The name of this utility function may not be clear (maybe `conditionalTooltip()`).